### ISSUE TITLE
xi=True produces the same effect as xi=False

### ISSUE THREAD

**Issue by jrmims** [LGSInnovations/sigplot#33]
`plot.change_settings(xi=True)` is the same as `plot.change_settings(xi=False)`

**Comment by jrmims** [comment]
It looks like xi was designed to be treated as a flag. https://github.com/LGSInnovations/sigplot/blob/master/js/sigplot.js#L6627

It doesn't appear to check for a value of true or false, just that the flag was used.

**Comment by mrecachinas** [comment]
@maihde This seems like the correct approach to me. Are there any reasons against this?

**Comment by sterre** [comment]
Is the idea that you could say `true` or `false` to force a color scheme, and no argument to toggle?

**Comment by jrmims** [comment]
I'm up for anything. I mostly wanted to know the intent of the `xi` parameter. I only ask because the jupyter-sigplot binder uses `xi=True` in an example, and I noticed `xi=False` still inverts the colors. Maybe it is just user error in the jupyter-sigplot binder?

**Comment by maihde** [comment]
The history of SigPlot's `xi` goes back to the MIDAS graphics library and it's use of `\XI` as a switch where no value is provided.  To maintain compatibility with the MIDAS command line, the plot constructor supported `xi` as an option.  The SigPlot code was implemented this way:

```
if (o.xi !== undefined) {
  mx.invertbgfg(Mx);
} 
```

This was done because calling the constructor with `xi: False` is redundant since that is the same as leaving `xi` out.  That said, I'm going to change to code so that `xi: False` doesn't cause the colors to invert.  This change I made should fix the issue for you in jupyter SigPlot.

FWIW, I made `change_settings` use a more intuitive name `invert`.    The `invert` settings supports three values: null (toggle), true (white background), and false (black background).  Sadly, the `xi` flag is not supported via `change_settings` and `invert` is not supported via constructor.  As part of the settings refactor, all of these inconsistencies will (hopefully) be resolved.
